src/core/perf/config/index.ts

```diff
@@ -11,32 +11,46 @@
  * @packageDocumentation
  */
 
-import type { PerfConfig, PerfTimerConfig, PerfPredicates, PerfGroupFilters } from 'core/perf/config/interface';
 import { GROUPS } from 'core/perf/const';
 import engines, { PerfTimerEngine } from 'core/perf/timer/engines';
 
+import { EXCLUDE } from 'core/perf/config/const';
+
+import type {
+
+	PerfConfig,
+	PerfTimerConfig,
+	PerfPredicates,
+	PerfGroupFilters,
+	PerfIncludeFilter
+
+} from 'core/perf/config/interface';
+
 export * from 'core/perf/config/interface';
 
 /**
- * Returns instance of the timer engine, defined in the performance config
+ * Returns an instance of the timer engine that defined in the performance config
  * @param config - performance config
  */
 export function getTimerEngine(config: PerfTimerConfig): PerfTimerEngine {
 	return engines[config.engine];
 }
 
 /**
- * Creates filter's predicates for every group
- * @param filters - filters from performance config
+ * Create filter predicates for every group
```

Comment:
Глагол действия метода должен быть в 3-м числе

```suggestion
 * Creates filter predicates for every group
```